CONTRIBUTION-GUIDELINES.md

```diff
@@ -1,4 +1,4 @@
-# Contributing
+# Contribution guidelines
```

Comment:
Slightly unsure about renaming this file – `CONTRIBUTING.md` is [a convention used by most open source projects](https://en.wikipedia.org/wiki/Contributing_guidelines). What do you think about changing the title whilst keeping the existing filename?

Reply:
Ah didn't know that - thanks. Yeah, let's keep the original title!